Compilation Errors: Cannot Find derive macro Parser and clap Attribute

I encountered compilation errors while trying to install `coffee_httpd`. The errors seem to be related to the use of the **`clap`** crate and the **`Parser`** trait. 

Command used: ```cargo install --path coffee_httpd```

Below is a summary of the errors:
```
error: cannot find derive macro `Parser` in this scope
error: cannot find attribute `clap` in this scope
error[E0599]: no function or associated item named `parse` found for struct `HttpdArgs` in the current scope

error: could not compile `coffee_httpd` (bin "coffee_httpd") due to 10 previous errors; 1 warning emitted
error: failed to compile `coffee_httpd v0.1.0 (/home/royalpinto007/coffee/coffee_httpd)`, intermediate artifacts can be found at `/home/royalpinto007/coffee/target`
```

--- a/coffee_httpd/Cargo.toml
+++ b/coffee_httpd/Cargo.toml
@@ -13,7 +13,7 @@ path = "src/main.rs"
 
 [dependencies]
 actix-web = "4"
-clap = "4.1.11"
+clap = { version = "4.1.11", features = ["derive"] }
 tokio = { version = "1.22.0", features = ["sync"] }
 coffee_core = { path = "../coffee_core" }
 coffee_lib = { path = "../coffee_lib", features = ["open-api"] }

